### PR TITLE
1 basic infra setup

### PR DESCRIPTION
**Pull request by phaus**
## Summary by CodeRabbit

- **New Features**
    - Updated web page setup for the "Bullshit Memory" application with user authentication and session management.
    - Added configurations for Live Server and Git ignore limit warning.

- **Documentation**
    - Enhanced project title, formatting, and content in README.
    - Included local development setup and production deployment instructions in README.

- **Chores**
    - Adjusted `.gitignore` for project-specific needs.

### CODE REVIEW

File: config.js
Comment (curiosdevcookie) [inline config.js:1]: localhost doesn't work for me, only 127.0.0.1

**Review by curiosdevcookie — COMMENTED** [review]
I will make some changes as well.

File: utils.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    var expireDate = new Date(now.getTime() + expires);
+    document.cookie = key + "=" + value + "; expires=" + expireDate.toGMTString() + ";";
+}
+
+const clearCookie = function(key) {
+    var now = new Date();
+    var expires = 1000 * 60 * 10;
+    var expireDate = new Date(now.getTime() - expires);
+    document.cookie = key + "=; expires=" + expireDate.toGMTString() + ";";
+}
```
[the remark this answers is not in the document]
Comment (curiosdevcookie) [reply utils.js:26]: This seems sensible to me.

File: config.js
Comment (phaus) [reply config.js:1]: Fine for me.
Then we have to update the configuration in appwrite & github to use `127.0.0.1` as a hostname.
And also the liveview config in vscode, I guess?

**Pull request merged**